solutions/security/endpoint-response-actions.md

```diff
@@ -221,7 +221,7 @@ You can follow this with the `execute` response action to upload and run scripts
 
 
 ::::{note}
-The default file size maximum is 25 MB, configurable in [`kibana.yml`](/deploy-manage/stack-settings.md) with the `xpack.securitySolution.maxUploadResponseActionFileBytes` setting. You must enter the value in bytes (the maximum is `104857600` bytes, or 100 MB).
+The default file size maximum is 25 MB, configurable in [`kibana.yml`](/deploy-manage/stack-settings.md) with the `xpack.securitySolution.maxUploadResponseActionFileBytes` setting. You must enter the value in bytes ({applies_to}`stack: removed 9.3`{applies_to}`serverless: removed` the maximum is `104857600` bytes, or 100 MB).
```

Comment:
This one is curious. I'm not sure I understand what this is saying. Is it that the default was 100MB and is now 25 MB?